### DISCUSSION TITLE
Search box as in Sequel Pro?

### DISCUSSION THREAD

**Discussion by dbaroncelli** [Sequel-Ace/Sequel-Ace#790] [Q&A] [answered]
In Sequel Pro there was the search box, which was really useful
<img width="971" alt="Schermata 2021-01-15 alle 00 17 40" src="https://user-images.githubusercontent.com/5320104/104660609-31dd4f00-56c7-11eb-9a52-685452798a92.png">

On Sequel Ace, I cannot find it.
Is there a way to enable it?

**Comment by Jason-Morcos** [comment, marked as answer]
Still there. Go to table->filter content or use the keyboard shortcut cmd+F to pop up the filters.

**Comment by dbaroncelli** [comment]
Hi Jason, thanks, I found it!
is there a way to leave it always there (as it used to be), without having to enable it each time?

**Reply by icealec** [reply]
HI. I would also like this feature
